pkgs/applications/networking/mailreaders/thunderbird/betterbird.diff

```diff
@@ -0,0 +1,44 @@
+--- a/12-feature-linux-systray-example.patch
++++ b/12-feature-linux-systray-example.patch
+@@ -8,18 +8,15 @@ diff --git a/third_party/appindicator/Makefile b/third_party/appindicator/Makefi
+ new file mode 100644
+ --- /dev/null
+ +++ b/third_party/appindicator/Makefile
+-@@ -0,0 +1,34 @@
++@@ -0,0 +1,31 @@
+ +# Code from https://github.com/AyatanaIndicators/libayatana-appindicator
+ +# and related repositories.
+ +# See https://github.com/AyatanaIndicators/libayatana-appindicator/issues/46 for build instructions.
+ +# You need: sudo aptitude install libdbusmenu-gtk3-dev
+ +
+-+CFLAGS=`pkg-config --cflags gtk+-3.0 glib-2.0` \
+-+  -I/usr/include/libdbusmenu-glib-0.4/ \
+-+  -I/usr/include/libdbusmenu-gtk3-0.4/ \
+-+  -I/usr/include/glib-2.0
+++CFLAGS=`pkg-config --cflags dbusmenu-gtk3-0.4` \
```

Comment:
Best practice is not relying on propagation for direct dependencies. It would break if ever the package moved some of the dependencies from `Requires` to `Requires.private` (not that I think it will ever happen in the case of a ~~dead~~ Ubuntu-maintained package like this one)

```suggestion
++CFLAGS=`pkg-config --cflags gtk+-3.0 glib-2.0 dbusmenu-gtk3-0.4` \
```